### PR TITLE
[KEYCLOAK-11079] CrossDC server tests broken on Travis

### PR DESCRIPTION
**Pull request by abstractj**
- Only run crossdc-server tests if the event type is a cron job

This change is not the definitive solution, but allows us to stabilize
the tests on master branch

### CODE REVIEW

**Review by mposolda — APPROVED** [review]
Approving assuming that this will be temporary and we will have JIRA with "critical" or "blocker" priority to revert it back when possible. Thing is, that we don't have cross-dc tests even in pipeline, so now there can be a bigger chance to introduce some regressions...

**Comment by hmlnarik** [comment]
IDK yet. It seems that the tests are really broken since they fail consistently. Let me check it more

**Comment by abstractj** [comment]
@hmlnarik np, I will hold the merge until you approve it. Just keep in mind that this change is just a temporary thing in an attempt to stabilize things, not the definitive solution.

**Review by hmlnarik — APPROVED** [review]
Approving since the tests pass locally. Apparently a Travis - specific thing. Perhaps we should prioritize cross-dc testing internally?

**Comment by abstractj** [comment]
> Approving since the tests pass locally. Apparently a Travis - specific thing. Perhaps we should prioritize cross-dc testing internally?

+1 on this

**Pull request merged**
